src/reformat/mod.rs

```diff
@@ -17,38 +17,38 @@ pub mod libyuv {
     use crate::*;
 
     pub fn yuv_to_rgb(_image: &image::Image, _rgb: &mut rgb::Image) -> AvifResult<bool> {
-        return Err(AvifError::NotImplemented);
+        Err(AvifError::NotImplemented)
```

Comment:
Is the last statement taken as the return?

Reply:
yep